docs/README.md

```diff
@@ -52,77 +40,27 @@ curl --request POST \
      '
 ```
 
-- Store Menu Updates: this records contains the items to be updated for each store:
-  The items list can be empty or contain the products with the initial information.
-
-```
-curl --request POST \
-     --url https://app.io.vtex.com/vtex.glovo-integration/v0/vtexspain/clerkiofeed/_v/glovo/menu-updates/save \
-     --header 'VtexIdClientAutCookie: {authToken}' \
-     --data '
-        {
-            "storeId": string,
-            "glovoStoreId": string,
-            "items": [
-                {
-                    id: string,
-                    price: number,
-                    available: boolean
-                }
-            ]
-        }
-     '
-```
-
-- Product records: this records are created automatically when receiving a catalog update for the product if they don't exist. Alternatively, this can be created manually:
-
-```
-curl --request POST \
-     --url https://app.io.vtex.com/vtex.glovo-integration/v0/vtexspain/clerkiofeed/_v/glovo/product-record/save \
-     --header 'VtexIdClientAutCookie: {authToken}' \
-     --data '
-        {
-            "storeId": string,
-            "skuId": string,
-            "data":
-                {
-                    id: string,
-                    price: number,
-                    available: boolean
-                }
-
-        }
-     '
-```
-
-4. The menu on Glovo can be updated completely or partially by using this endpoints:
+5. The catalog of products on Glovo can be updated completely or partially by using this endpoints:
 
 - Complete update
 
 ```
 curl --request POST \
-     --url https://clerkiofeed--vtexspain.myvtex.com/_v/glovo/menu/update-all \
+     --url https://{{account}}.myvtex.com/_v/glovo/menu/update-all \
      --header 'VtexIdClientAutCookie: {authToken}' \
 ```
 
 - Partial update
 
 ```
 curl --request POST \
-     --url https://clerkiofeed--vtexspain.myvtex.com/_v/glovo/menu/update-partial \
+     --url https://{{account}}.myvtex.com/_v/glovo/menu/update-partial \
```

Comment:
let's add that the partial update requires the use of the Affiliate configured by SC